var/spack/repos/builtin/packages/steps/package.py

```diff
@@ -42,13 +46,16 @@ class Steps(CMakePackage):
     depends_on("py-scipy", type=("build", "test"))
     depends_on("py-matplotlib", type=("build", "test"))
     depends_on("python")
-
+    depends_on('git', type='build', when='+codechecks')
```

Comment:
IIRC, git is prerequisite for spack. So no need to add it here.

Reply:
dependency removed in fda07d7